apps/health_campaign_field_worker_app/lib/blocs/beneficiary_registration/beneficiary_registration.dart

```diff
@@ -200,6 +200,11 @@ class BeneficiaryRegistrationBloc
               ),
             ),
           );
+          final cycleIndex = household.additionalFields!.fields
```

Comment:
null check this can become null as well right??

Reply:
Have added the null checks where ever required